src/Features/DCBreadcrumb.php

```diff
@@ -171,10 +262,32 @@ private function getItemEnclosurePosition($itemtype, $items_id)
             $position = $ien->getField('position');
         }
 
-        $position = "&nbsp;" . sprintf(__('(U%1$u)'), $position);
+        $position = sprintf(__('(U%1$u)'), $position);
         return $position;
     }
 
+    /**
+     * get item location from Enclosure
+     *
+     * @param string  $itemtype Item type
+     * @param integer $items_id Item ID
+     *
+     * @return string
+     */
+    private function getItemLocation($itemtype, $items_id)
```

Comment:
a suggestion here, this method seems not related to DCBreadcrumb class and doesn't require any `$this`.
I suggest to move it to Location class as private method.
Like `Location::getForItem(CommonDBTM $item)`  (avoid itemtype, items_id parameter when you the item initialy)